### ISSUE TITLE
only and ignore options are ignored with babel-core v5.5.x

### ISSUE THREAD

**Issue by jwhitfieldseed** [babel/babelify#88]
I am trying to gnore files in a specific directory. That stopped working after an `npm install` today. The babelify version did not change, but its babel-core dependency updated from v5.4.x to v5.5.x.

Browserify config in package.json:

````json
"browserify": {
  "transform": [
    ["babelify", {
      "optional": ["runtime"],
      "ignore": ["./public/js/lib"]
    }]
  ]
}
````

I am using babelify v6.6.1 and browserify v10.2.3.

When babelify uses `babel-core` @v5.4.7, files in `./public/js/lib` are ignored as expected. With `babel-core` @v5.5.x they are not ignored. I have tried replacing the dir path with an absolute path to a single file, which also had no effect. The `only` option also appears to have no effect in v5.5.x.

The babel CLI `--only` and `--ignore` flags work as expected in v5.5.x, which leads me to think it may be a babelify compat issue.
 
N.B. I'm running with grunt-browserify. I see the effect when the config is in `Gruntfile.js`, `package.json` as above, or passed directly to `babelify.configure`.

**Comment by thaiat** [comment]
similar issue here : https://github.com/babel/babel/issues/1693

**Comment by jwhitfieldseed** [comment]
Seems like this should be fixed byhttps://github.com/babel/babel/commit/b0442d0784299984cde50d97a313e0cd3a7b3d3a.

**Comment by sebmck** [comment]
Fixed as of Babel 5.5.4, thanks! :)
